README.md

```diff
@@ -0,0 +1,33 @@
+# hash-verifier
+
+hash verification using a nonce & verifying the hmac from the URL for a laravel application
+
+## Installation
+
+```bash
+composer require dpc/hash-verifier
+```
+
+## Usage
+
+TODO: Write usage instructions
```

Comment:
can you attempt this based on the example I have given? If not I will add it later